schema/Retail/Cancel/cancel.ts

```diff
@@ -53,7 +53,7 @@ export const cancelSchema = {
         },
         timestamp: {
           type: 'string',
-          format: 'date-time',
+          format: 'rfc3339-date-time',
```

Comment:
Is there any regex defined for this? We have to make sure that it should be in UTC format too, with exactly the following pattern YYYY-MM-DDTHH:MM:SS.MSSZ